### ISSUE TITLE
1 issue and 2 feature requests

### ISSUE THREAD

**Issue by ecrousphilker1** [timer-machine/timer-machine-android#90]
Issue 1: Doesn't lower the sound of other media when step voices are still on going. (or it doesn't lower the sound long enough?)

Request 1: To add pop a picture along the "fullscreen" option during the steps. It would help especially during workouts when I don't exactly remember the proper form.

Request 2: to add other open source voices. Sometimes it's just weird especially when there are longer texts to be transcripted

**Comment by DeweyReed** [comment]
You can control how the app's audio interacts with other apps' audio by changing the "Audio Focus Type" value in the app settings. Change it to "None" to play all sounds simultaneously.

The image feature is under development, as mentioned in issue #30.

What kind of voices would you like to be included in the app? Copyright can be complicated.

**Comment by ecrousphilker1** [comment]
1. I'm sorry, I wasn't clear. I actually liked the lower sound feature. Now that I've done many workouts with it I realized that it respects the notifiers more than the steps ie. My music lowers when notifier is speaking goes up again (sometimes not) and lowers when the countdown started. and then when it proceeds to the step, the step voice (typically just the step name) is ignored and the music is still is max volume, it leaves me confused and forces me to get hands on my phone back.

2. About the voices: I didn't realize it was complicated. Other free timers like O'Coach and Just Another Workout Timer has multiple voice options. I'm using this timer because it has more features I didn't even know I needed and a lot more stable. The problem I'm having with the current voice is it speaks too fast and pronounces words that are hard to follow. Hard for non-native speakers.

Thank you

**Comment by DeweyReed** [comment]
Regarding the lower sound feature, the app doesn't specially treat notifiers. Could you give an example that can reproduce the problem?

This app's voice feature entirely relies on the system's Text-to-speech engine. Therefore, the app has little control over the quality of the voice. You can search "Text-to-speech" in the system settings; some engines provide options to adjust its speed and tone.

**Comment by ecrousphilker1** [comment]
Here's my on of timers: [https://docs.google.com/document/d/1fru3igkFHBWpI3JeneV1R_32C2l_tL9Mieam9dZzdIc/edit?usp=drivesdk]()

I tried it on YT Music, YT, VLC. Sometimes it lowers the other media for too long even after timer voices, sometimes it doesn't lower at all. As I'm writing I'm thinking maybe because of the countdown? 

God I had a feeling it was the system's I didn't it was possible. Thank you!

**Comment by ecrousphilker1** [comment]
Oh yeah it was the countdown

**Comment by DeweyReed** [comment]
Thanks for your explanation! I've understood the situation. It's complicated because many steps and various reminders are involved. I need to come up with a general solution for the audio focus.

**Comment by ecrousphilker1** [comment]
I found a pretty good work around maybe it's a good place to start. I just put empty notifier for 1 second in between steps. Because of that I got another variable request for the voice. Make the next step name to be an actual step and not a notifier. Its useful if you want to be reminded of what's coming next after a short rest.
